src/commons/locations.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Seattle": "Seattle",
+    "San_Francisco": "San Francisco / Bay",
```

Comment:
Why `/ Bay` ? Also I think the RHS should be snake cased because that's what we are passing into template.